packages/showcases/svelte/stories/components/navigation/VtmnNavbar/VtmnNavbar.stories.svelte

```diff
@@ -0,0 +1,159 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import {
+    argTypes,
+    parameters,
+  } from '@vtmn/showcase-core/csf/components/navigation/navbar.csf';
+  import { VtmnButton, VtmnNavbar, VtmnSearch } from '@vtmn/svelte';
+</script>
+
+<Meta
+  title="Components / Navigation / VtmnNavbar"
+  component={VtmnNavbar}
+  {argTypes}
+  {parameters}
+/>
+
+<Story name="Overview" let:args>
+  <div style="width: 800px; display: flex; justify-content: center">
+    <VtmnNavbar {...args}>
+      <svelte:fragment slot="logo">
+        <a class="vtmn-navbar_logo" href="#">
```

Comment:
Question : 
I think the `<a class="vtmn-navbar_logo" href="#">` should be inside the component `VtmnNavbar`.

@lauthieb , for all logo, if we click on it, will it always have to go back to the href '#' ?

Reply:
@lauthieb what do you think about it ?

Reply:
@Tlahey @jb-thery totally agree. Logo should be inside this component and it will always redirect to the homepage, so I think "#" in `href` is correct.